packages/manager/modules/octavia-load-balancer/src/load-balancers/load-balancer/pools/detail/template.html

```diff
@@ -11,14 +11,14 @@
     >
         <span data-translate="load_balancer_pools_detail_info_tab_title"></span>
     </oui-header-tabs-item>
-    <oui-header-tabs-item
+    <!--<oui-header-tabs-item
         data-href="{{:: $ctrl.healthMonitorPoolsDetailLink }}"
         data-active="$ctrl.currentActiveLink().startsWith($ctrl.healthMonitorPoolsDetailLink)"
     >
         <span
             data-translate="load_balancer_pools_detail_health_monitor_tab_title"
         ></span>
-    </oui-header-tabs-item>
+    </oui-header-tabs-item>-->
```

Comment:
No need to keep it in comment

Reply:
OK. I removed it and cleaned the routing file as well.